client/dist/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+
+<head>
```

Comment:
Can you add the metadata for charset and viewport? This is available on MDN
Defines default characters
<meta charset="UTF-8">
Defines window size for desktop and mobile
<meta viewport=???>